README.adoc

```diff
@@ -10,7 +10,7 @@ eval $(minikube docker-env)
 ```
 
 * A working Go environment, with a clone of ericbottard's customized fabric8io/kubernetes-model.
-
++
```

Comment:
is this `+` intentional? (I had removed it earlier I think)

Reply:
Nope, certainly a rebase glitch

Reply:
Actually I'm sorry, YES it is intentional. It fuses the code snippet with the list item (otherwise it is not correctly indented)

Reply:
nice - I didn't know about that